benchmark/README.md

```diff
@@ -1,147 +1,276 @@
-# Node.js core benchmark tests
+# Node.js core benchmark
 
-This folder contains benchmark tests to measure the performance for certain
-Node.js APIs.
+This folder contains benchmarks to measure the performance of the Node.js APIs.
+
+## Table of Content
+
+* [Prerequisites](#prerequisites)
+* [Running benchmarks](#running-benchmarks)
+ * [Running individual benchmarks](#running-individual-benchmarks)
+ * [Running all benchmarks](#running-all-benchmarks)
+ * [Comparing node versions](#comparing-node-versions)
+ * [Comparing parameters](#comparing-parameters)
+* [Creating a benchmark](#creating-a-benchmark)
 
 ## Prerequisites
 
-Most of the http benchmarks require [`wrk`][wrk] and [`ab`][ab] (ApacheBench) being installed.
-These may be available through your preferred package manager.
+Most of the http benchmarks require [`wrk`][wrk] to be installed. It may be
+available through your preferred package manager. If not `wrk` can be built
+[from source][wrk] via `make`.
+
+The R scripts uses `ggplot2` and `plyr`, you can install them using from the
+R REPL.
 
-If they are not available:
-- `wrk` may easily be built [from source][wrk] via `make`.
-- `ab` is sometimes bundled in a package called `apache2-utils`.
+```
+$ R
+$ install.packages("ggplot2")
+$ install.packages("plyr")
+```
 
 [wrk]: https://github.com/wg/wrk
-[ab]: http://httpd.apache.org/docs/2.2/programs/ab.html
 
-## How to run tests
+## Running benchmarks
+
+### Running individual benchmarks
 
-There are three ways to run benchmark tests:
+This can be useful for debugging a benchmark or doing a quick performance
+measure. But it does not provide the statistical information to make any
+conclusions about the performance.
 
-### Run all tests of a given type
+individual benchmarks can be executed by simply executing the benchmark script
```

Comment:
search and replace part is the same? Looked it up in oxford, it appears `individual` is the correct spelling.